buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java

```diff
@@ -280,6 +280,28 @@ protected ByteBuf newDirectBuffer(int initialCapacity, int maxCapacity) {
         return toLeakAwareBuffer(buf);
     }
 
+    public static int getDefaultNumHeapArena() {
+        return DEFAULT_NUM_HEAP_ARENA;
+    }
+    public static int getDefaultNumDirectArena() {
+        return DEFAULT_NUM_DIRECT_ARENA;
+    }
+    public static int getDefaultPageSize() {
+        return DEFAULT_PAGE_SIZE;
+    }
+    public static int getDefaultMaxOrder() {
+        return DEFAULT_MAX_ORDER;
+    }
+    public static int getDefaultTinyCacheSize() {
+        return DEFAULT_TINY_CACHE_SIZE;
+    }
+    public static int getDefaultSmallCacheSize() {
+        return DEFAULT_SMALL_CACHE_SIZE;
+    }
+    public static int getDefaultNormalCacheSize() {
+        return DEFAULT_NORMAL_CACHE_SIZE;
+    }
```

Comment:
we not use get* prefixes in netty. Please remove these. Also please add javadocs.